src/mame/drivers/magicard.cpp

```diff
@@ -836,82 +665,82 @@ static INPUT_PORTS_START( lucky7i )
 	PORT_BIT( 0x0020, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_NAME("Win Plan Scroll/Collect")
 	PORT_BIT( 0x0040, IP_ACTIVE_LOW, IPT_POKER_HOLD5 ) PORT_NAME("Einsatz")
 	PORT_BIT( 0x0080, IP_ACTIVE_LOW, IPT_START1 ) PORT_NAME("Start/Gamble")
-	PORT_BIT( 0x0100, IP_ACTIVE_LOW, IPT_SERVICE2 ) PORT_NAME("Rental Book Keeping")
+	PORT_BIT( 0x0100, IP_ACTIVE_LOW, IPT_GAMBLE_BOOK ) PORT_NAME("Rental Book Keeping")
 
 	PORT_BIT( 0x8000, IP_ACTIVE_LOW, IPT_UNUSED )
 
 	PORT_MODIFY("SW1")
-	PORT_DIPNAME( 0x80, 0x80, "Hopper" )					PORT_DIPLOCATION("DIP 1:1")
+	PORT_DIPNAME( 0x80, 0x80, "Hopper" )                    PORT_DIPLOCATION("DIP 1:1")
 	PORT_DIPSETTING(    0x80, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
 
-	PORT_DIPNAME( 0x40, 0x40, "Hopper-Wert" )				PORT_DIPLOCATION("DIP 1:2")
+	PORT_DIPNAME( 0x40, 0x40, "Hopper-Wert" )               PORT_DIPLOCATION("DIP 1:2")
 	PORT_DIPSETTING(    0x40, "10" )
 	PORT_DIPSETTING(    0x00, "5" )
 
-	PORT_DIPNAME( 0x08, 0x08, "Munzer1" )					PORT_DIPLOCATION("DIP 1:5")
+	PORT_DIPNAME( 0x08, 0x08, "Munzer1" )                   PORT_DIPLOCATION("DIP 1:5")
 	PORT_DIPSETTING(    0x08, "10" )
 	PORT_DIPSETTING(    0x00, "5" )
 
-	PORT_DIPNAME( 0x04, 0x04, "Munzer2" )					PORT_DIPLOCATION("DIP 1:6")
+	PORT_DIPNAME( 0x04, 0x04, "Munzer2" )                   PORT_DIPLOCATION("DIP 1:6")
 	PORT_DIPSETTING(    0x04, "10" )
 	PORT_DIPSETTING(    0x00, "5" )
 
-	PORT_DIPNAME( 0x02, 0x02, "Remote 1" )					PORT_DIPLOCATION("DIP 1:7")
+	PORT_DIPNAME( 0x02, 0x02, "Remote 1" )                  PORT_DIPLOCATION("DIP 1:7")
 	PORT_DIPSETTING(    0x02, "100" )
 	PORT_DIPSETTING(    0x00, "10" )
 
-	PORT_DIPNAME( 0x20, 0x20, "Remote 2" )					PORT_DIPLOCATION("DIP 1:3")
+	PORT_DIPNAME( 0x20, 0x20, "Remote 2" )                  PORT_DIPLOCATION("DIP 1:3")
 	PORT_DIPSETTING(    0x20, "100" )
 	PORT_DIPSETTING(    0x00, "50" )
 
-	PORT_DIPNAME( 0x10, 0x10, DEF_STR( Unknown ) )			PORT_DIPLOCATION("DIP 1:4")
+	PORT_DIPNAME( 0x10, 0x10, DEF_STR( Unknown ) )          PORT_DIPLOCATION("DIP 1:4")
 	PORT_DIPSETTING(    0x10, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
 
-	PORT_DIPNAME( 0x01, 0x01, DEF_STR( Unknown ) )			PORT_DIPLOCATION("DIP 1:8")
+	PORT_DIPNAME( 0x01, 0x01, DEF_STR( Unknown ) )          PORT_DIPLOCATION("DIP 1:8")
 	PORT_DIPSETTING(    0x01, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
 
 	PORT_BIT( 0x8000, IP_ACTIVE_LOW, IPT_UNUSED )
 	PORT_BIT( 0x0200, IP_ACTIVE_LOW, IPT_UNUSED )
-	PORT_BIT( 0x0100, IP_ACTIVE_LOW, IPT_BUTTON6 ) PORT_NAME("Attendant Collect") PORT_CODE(KEYCODE_A)
+	PORT_BIT( 0x0100, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_NAME("Attendant Collect") PORT_CODE(KEYCODE_A)
```

Comment:
Is this a control the attendant uses when the player wants to collect their winnings?  If it is, you can use `IPT_GAMBLE_KEYOUT` for it.  (If a gambling machine has a player control for collecting winnings, that’s `IPT_GAMBLE_PAYOUT`; if it has an attendant control for collecting winnings, that’s `IPT_GAMBLE_KEYOUT`).